docs/content/API-Reference/SQL-API.mdx

```diff
@@ -369,6 +369,29 @@ module.exports = {
 };
 ```
 
+## Security Context (Row-Level Security)
+
+Cube.js provides security context functionality, which can be used in the
+[Dynamic Schema Creation][ref-dynamic-schemas] or [`queryRewrite`][ref-config-queryrewrite] property in your [`cube.js`
+configuration file][ref-config-js].
+
+By default, SQL API uses current user's security context, but there is way how to change it.
+
+First, you need to define what user is allowed to change security context:
+
+```
+CUBEJS_SQL_SUPERUSER=admin
```

Comment:
We should add `CUBEJS_SQL_SUPERUSER` to https://cube.dev/docs/reference/environment-variables

Reply:
> I didn't find a section for it. Should I create a new one?

That's a great point, could you create a new 'SQL API' section? Thank you 😀

Reply:
✅

Reply:
Amazing, thank you